edb/pgsql/compiler/config.py

```diff
@@ -48,6 +49,14 @@ def compile_ConfigSet(
     result: pgast.BaseExpr
 
     if op.scope is qltypes.ConfigScope.INSTANCE and op.backend_setting:
+        if not (
+            ctx.env.backend_runtime_params.instance_params.capabilities
+            & pgparams.BackendCapabilities.CONFIGFILE_ACCESS
+        ):
+            raise errors.UnsupportedBackendFeatureError(
+                "configuring backend parameters via CONFIGURE INSTANCE"
+                " is not supported by the current backend"
```

Comment:
Do we have something like "backend name" or "backend type" in our system? E.g. it would be cool to explicitly say "this isn't supported on RDS" or "... on Stolon"

Reply:
We don't.  We do feature-based inference of backend capabilities, but, strictly speaking, there isn't a way to get a good "title" for the host kind.